spec/bot_spec.rb

```diff
@@ -218,6 +219,26 @@
         )
         handle_pull_request
       end
+
+      context "when magic_qa_keyword has not been given" do
```

Comment:
I think that you meant `max_description_chars`.

Reply:
Good catch 👍